eip-5484.md

```diff
@@ -0,0 +1,104 @@
+
+This EIP envisions Soulbound Tokens as specialized NFTs that will play the roles of credentials, credit records, loan histories, memberships, and many more. In order to provide the flexibility in these scenarios, Soulbound Token must have a mission specific burn authorization and a way to distinguish itself from regular ERC-721 tokens.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+- The token MUST implement the following interfaces:
+
+  1. [EIP-165](./eip-165.md)’s `ERC165` (`0x01ffc9a7`)
+  1. [EIP-721](./eip-721.md)’s `ERC721` (`0x80ac58cd`)
+
+- `burnAuth` SHALL be presented to receiver before issurance. 
+- `burnAuth` SHALL be Immutable after issurance.
+- `burnAuth` SHALL be the sole factor that determines which party has the rights to burn token.
+- The issuer SHALL present token metadata to the receiver and acquire receiver's signature before issurance.
+- The issuer SHALL NOT change metadata after issurance.
+
+### Contract Interface
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface IERC3000 {
+    /// A guideline to standardlize burn-authorization's number code
+    enum BurnAuth {
+        IssuerOnly,
+        OwnerOnly,
+        Both,
+        Neither
+    }
+
+    /// @notice Emitted when a Soulbound Token is issued.
+    /// @dev This emit is an add-on to nft's transfer emit in order to distinguish sbt 
+    /// from vanilla nft while providing backward compatibility.
+    /// @param from The issuer
+    /// @param to The receiver
+    /// @param tokenId The id of the issued token
+    event Issued (
+        address indexed from,
+        address indexed to,
+        uint256 indexed tokenId,
+        BurnAuth burnAuth
+    );
+
+    /// @notice provides burn authorization of the token id.
+    /// @dev unassigned tokenIds are invalid, and queries do throw
+    /// @param tokenId The identifier for a token.
+    function burnAuth(uint256 tokenId) external view returns (BurnAuth);
+}
+```
+
+## Rationale
+### SBT as an extension to IERC721
+We believe that Soulbound Token serves as a specialized subset of the existing ERC721 tokens. The advantage of such design is seamless compatibility of Soulbound Token with existing NFT services. Service providers can treat SBTs like NFTs and do not need to make drastic changes to their existing codebase. 
+
+### Non-Transferable 
+One problem with current Soulbound Token implementations that extend from [EIP-721](./eip-721.md) is that all transfer implementations throw errors. A much cleaner approach would be, while transfer functions still throw, thrid-parties can check beforehand if the contract implements Soulbound interface. And if so, they can avoid calling transfer. 
```

Comment:
```suggestion
One problem with current Soulbound Token implementations that extend from [EIP-721](./eip-721.md) is that all transfer implementations throw errors. A much cleaner approach would be for transfer functions to still throw, but also enable third parties to check beforehand if the contract implements the soulbound interface to avoid calling transfer. 
```